### PR TITLE
fix(48053) Description of tsBuildInfoFile is misleading in generated tsconfig.json

### LINKED ISSUE #48053: Description of `tsBuildInfoFile` is misleading in generated tsconfig.json
# Bug Report

### 🔎 Search Terms

tsconfig tsBuildInfoFile Specify the folder

### 🕗 Version & Regression Information

4.5.5, 4.7.0-dev.20220227

### ⏯ Playground Link

N/A

### 💻 Code

After running `tsc --init` the generated `tsconfig.json` includes the following line:

```js
  // "tsBuildInfoFile": "./",                          /* Specify the folder for .tsbuildinfo incremental compilation files. */
```

### 🙁 Actual behavior

The description says "Specify the **folder**" and the sample value also indicates that you specify a folder for this option.

However, specifying a folder would result in a following error when running `tsc`:

```
error TS5033: Could not write file '/path/to/folder': EISDIR: illegal operation on a directory, open '/path/to/folder'.
```

### 🙂 Expected behavior

Actually you need to specify the **file** .

[The official tsConfig doc](https://www.typescriptlang.org/tsconfig#:~:text=This%20setting%20lets%20you%20specify%20a%20file) also says “This setting lets you specify a **file**”.

 A fix would be, for example:

```js
  // "tsBuildInfoFile": "./.tsbuildinfo",                          /* Specify the path to .tsbuildinfo incremental compilation file. */
```

### PR DESCRIPTION
**Pull request by Blopaa**
Fixes #48053

### CODE REVIEW

**Comment by jablko** [comment]
I've now opened https://github.com/microsoft/TypeScript-Website/pull/2303 to also clarify the handbook.

**Review by andrewbranch — CHANGES_REQUESTED** [review]
@Blopaa thanks! This looks good to me, but you’ll need to [update the baselines](https://github.com/Microsoft/TypeScript/blob/main/CONTRIBUTING.md#managing-the-baselines) so tests pass.

**Comment by Blopaa** [comment]
tests still fail

**Comment by andrewbranch** [comment]
It looks like you need to run and update baselines again after merging with main. I’m guessing something else in the last 20 days touched these baselines in a way where the git merge line order was ambiguous.

**Review by sheetalkamat — APPROVED** [review]

File: tests/baselines/reference/tsConfig/Default initialized TSConfig/tsconfig.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
     /* Projects */
     // "incremental": true,                              /* Save .tsbuildinfo files to allow for incremental compilation of projects. */
     // "composite": true,                                /* Enable constraints that allow a TypeScript project to be used with project references. */
-    // "tsBuildInfoFile": "./",                          /* Specify the folder for .tsbuildinfo incremental compilation files. */
-    // "disableSourceOfProjectReferenceRedirect": true,  /* Disable preferring source files instead of declaration files when referencing composite projects. */
+    // "tsBuildInfoFile": "./.tsbuildinfo",              /* Specify the path to .tsbuildinfo incremental compilation files. */
```
Comment (sheetalkamat) [inline tests/baselines/reference/tsConfig/Default initialized TSConfig/tsconfig.json:8]: i think it should be `file` instead of `files`

Comment (Blopaa) [reply tests/baselines/reference/tsConfig/Default initialized TSConfig/tsconfig.json:8]: originally it was files, it must be something else, I did pull to main and ran `gulp baseline-accept` and it still failed.

Comment (andrewbranch) [reply tests/baselines/reference/tsConfig/Default initialized TSConfig/tsconfig.json:8]: You have to run the tests again before doing `gulp baseline-accept`

Comment (andrewbranch) [reply tests/baselines/reference/tsConfig/Default initialized TSConfig/tsconfig.json:8]: I agree the message should say `file`

**Review by andrewbranch — APPROVED** [review]

**Comment by andrewbranch** [comment]
Thanks!

**Pull request merged**